client/keys/export_test.go

```diff
@@ -31,7 +31,7 @@ func Test_runExportCmd(t *testing.T) {
 		kb.Delete("keyname1") // nolint:errcheck
 	})
 
-	path := sdk.GetConfig().GetFullFundraiserPath()
+	path := sdk.GetConfig().GetFullBip44Path()
```

Comment:
A bit pedantic, but Bip should be BIP

```suggestion
	path := sdk.GetConfig().GetFullBIP44Path()
```